signinapp/model.py

```diff
@@ -48,6 +47,13 @@ def canonical_name(name: str):
         return mk_hash(f"{m['first']} {m['last']}")
 
 
+def get_form_ids(model, prefix=False):
```

Comment:
`prefix` does not do at all what I was expecting.. Maybe `add_none_option` or something?